lighthouse-core/report/templates/report-template.html

```diff
@@ -177,3 +176,13 @@ <h1 class="menu__header-title">Lighthouse</h1>
     </footer>
   </section>
 </div>
+
+{{#if scripts }}
+<script>
+  window.lhresults = {{{lhresults}}};
```

Comment:
Ah, so to be clear, these `<script>`s (and therefore the `{{{}}}`) never see user-provided input. 

The flow is:
User clicks "open in viewer", the local page opens the online viewer (a page with [different markup](url)) and sends the `window.lhresults` JSON  to it. That page's `message` [listener](https://github.com/GoogleChrome/lighthouse/pull/1179/files#diff-fe42ad1345a50fa5e61842d4e35473b3R410) does the `replaceReportHTML`. That method then goes through our [normal Handlebars pipeline](https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-viewer/app/src/lighthouse-report-viewer.js#L152-L159) to generate the inner page content.